zipline/src/commonMain/kotlin/app/cash/zipline/internal/bridge/Endpoint.kt

```diff
@@ -143,18 +143,6 @@ class Endpoint internal constructor(
     return inboundServices.remove(name)
   }
 
-  @PublishedApi
-  internal fun remove(service: ZiplineService) {
```

Comment:
We don’t want this! (Also the `@PublishedApi` looks like a copy-paste mistake from the remove() overload we actually need.)